### PR TITLE
Sync with starter template

### PR DESCRIPTION
**Pull request by mickmister**
Fixes https://github.com/mattermost/mattermost-plugin-agenda/issues/52

### CODE REVIEW

File: webapp/tests/i18n_mock.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
[the remark this answers is not in the document]
Comment (mickmister) [reply webapp/tests/i18n_mock.json:1]: Please see my answer here https://github.com/mattermost/mattermost-plugin-github/pull/345#discussion_r471531420

**Review by hanzei — APPROVED** [review]
While I would like to see the latest changes from https://github.com/mattermost/mattermost-plugin-starter-template/pull/86 getting merged in this, the PR is already a great improvement in terms of tooling.

Heads up that there are merge conflicts to resolve.

**Comment by hanzei** [comment]
@DHaussermann These are only tooling updates. Maybe they can be tested in release testing of v0.2.1.

**Review by DHaussermann — APPROVED** [review]
Agreed @hanzei please merge.

LGTM!

**Pull request merged**